app/konnect/dev-portal/index.md

```diff
@@ -40,8 +40,26 @@ For developers:
 
 * [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform.
 
+### Contextual developer analytics 
+
+You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a customizable dashboard. 
```

Comment:
I think we should avoid saying "customizable" in this sentence. Readers may think that that they can add/remove charts from this dashboard (which is referred to as "customizable dashboard").